spec/factories/digital_instantiation.rb

```diff
@@ -8,7 +8,7 @@
     duration { '1:23:45' }
     file_size { '12354435234' }
     local_instantiation_identifier { ["1234"] }
-    digital_instantiation_pbcore_xml { Rack::Test::UploadedFile.new(Rails.root.join('spec/fixtures/sample_instantiation_valid.xml'), 'text/xml') }
+    digital_instantiation_pbcore_xml { File.open(Rails.root.join('spec/fixtures/sample_instantiation_valid.xml')) }
```

Comment:
The `digital_instantiation_pbcore_xml` is defined in `DigitalInstantiation` class with
```
mount_uploader :digital_instantiation_pbcore_xml, PbCoreInstantiationXmlUploader
```
I don't recall offhand which kind of object that this ends up being, but there 3 places where the code expects the object to have some method on it:
* `digital_instantiation_pbcore_xml.file`
* `digital_instantiation_pbcore_xml.file.file`
* `digital_instantiation_pbcore_xml.file.tempfile`

I think `File.open(...)` would have to return a similar object in order to work but not sure if it does.

Reply:
I just tried these three methods using a debugger locally: 

- `File.open` 
- `Rack::Test::UploadedFile.new` 
- `fixture_file_upload` 

All three result in `digital_instantiation_pbcore_xml` being an instance of `PbCoreInstantiationXmlUploader` with, as far as I can tell, identical behavior. 

@sephirothkod may have more info on this. I don't know much beyond "this wasn't working before, this change seems to make it work better"